install.sh

```diff
@@ -194,16 +200,16 @@ for package in $BOOTSTRAP_PACKAGES; do
   # This is really ugly, FIXME after #7082 is merged.
   if [[ "${package}" == "zstd" ]]; then
     if [[ "$ARCH" == "aarch64" ]]; then
-      urls+=("https://gitlab.com/api/v4/projects/26210301/packages/generic/${name}/${version}_armv7l/${name}-${version}-chromeos-armv7l.tar.xz")
+      urls+=("${PREBUILT_URL}/${name}/${version}_armv7l/${name}-${version}-chromeos-armv7l.tar.xz")
     else
-      urls+=("https://gitlab.com/api/v4/projects/26210301/packages/generic/${name}/${version}_${ARCH}/${name}-${version}-chromeos-${ARCH}.tar.xz")
+      urls+=("${PREBUILT_URL}/${name}/${version}_${ARCH}/${name}-${version}-chromeos-${ARCH}.tar.xz")
     fi
   else
     [[ "${package}" == "glibc_lib235" ]] && name="glibc_lib"
     if [[ "$ARCH" == "aarch64" ]]; then
-      urls+=("https://gitlab.com/api/v4/projects/26210301/packages/generic/${name}/${version}_armv7l/${name}-${version}-chromeos-armv7l.tar.zst")
+      urls+=("${PREBUILT_URL}/${name}/${version}_armv7l/${name}-${version}-chromeos-armv7l.tar.zst")
     else
-      urls+=("https://gitlab.com/api/v4/projects/26210301/packages/generic/${name}/${version}_${ARCH}/${name}-${version}-chromeos-${ARCH}.tar.zst")
+      urls+=("${PREBUILT_URL}/${name}/${version}_${ARCH}/${name}-${version}-chromeos-${ARCH}.tar.zst")
```

Comment:
Reduce `"${PREBUILT_URL}/${name}/${version}_${ARCH}/${name}-${version}-chromeos-${USER_SPACE_ARCH}.tar.${ARCHIVE_EXTENSION}"` to a signal variable and simply assign `ARCHIVE_EXTENSION` instead?

Reply:
And we don't need to assign ARCHIVE_EXTENSION, we can just use the binary_compression value in each package.

Reply:
(once this is rebased onto master, this review is no longer relevant)